navbar.php

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>STARBULLS</title>
+</head>
+
+<body>
```

Comment:
none of this is needed since its only being included as a snippet. This is likely where some of the perceived css issues are coming from.